index.html

```diff
@@ -2979,6 +2979,15 @@ <h2 id="notes">Notes</h2>
         </ul>
       </p>
 
+      <p id="relational-operator-note">
```

Comment:
It is OK to copy-paste another section and then edit, but you need to give the new section its own unique `id`. :-)